source/json/kernel-4.3/datacite_4.3_schema.json

```diff
@@ -16,18 +19,21 @@
                 "properties": {
                     "nameIdentifier": {"type": "string"},
                     "nameIdentifierScheme": {"type": "string"},
-                    "schemeURI": {"type": "string", "format": "uri"}
+                    "schemeUri": {"type": "string", "format": "uri"}
                 },
                 "required": ["nameIdentifier", "nameIdentifierScheme"]
             },
             "uniqueItems": true
         },
-        "affiliations": {
+        "affiliation": {
```

Comment:
This modification is not consistent with the references used below in the `creators` and `contributors` sections: 
```
                    "affiliations": {"$ref": "#/definitions/affiliations"},
```
I would keep the plural form, since the element is in array type (hence a list of affiliation items).

Reply:
I agree, however DataCite is using affiliation in their JSON API (e.g. https://github.com/datacite/schema/blob/master/source/json/kernel-4.3/example/datacite-example-full-v4.json and https://api.datacite.org/dois/10.22002/D1.1278). If the JSON schema uses affiliations you cannot submit valid JSON to DataCite.

Reply:
ok then, we should update the attribute name in the `creators` and `contributors` objects, not in the referenced definition.

Reply:
Makes sense, just a minute